### PR TITLE
DO-955 Logo link +  DO-956 Copyrights

### PR DESCRIPTION
**Pull request by Litsyshyn**
Added link to Landing on platform login page. 
https://razom.atlassian.net/browse/DO-955

Also according to DO-956, added Copyrights to platform login page. 
As i understand it should be changed, becuse page structure is incorrect(?). But if it acceptable as a temporary desicion, let's use it.
https://razom.atlassian.net/browse/DO-956

I was check everything few time, but anyway sure that there are something wrong, so if Copirights are unacceptable, i'll delete it.

### CODE REVIEW

File: src/components/pages/auth/LoginPage.jsx [hunk 1]
```diff
@@ -49,13 +49,15 @@ const LoginPage = ({ match, history, location }) => {
       <div className="block xl:grid grid-cols-2 gap-4">
         <div className="hidden xl:flex flex-col min-h-screen">
           <div className="-intro-x flex items-center pt-8">
-            <img alt="Data Ocean Logo" className="w-12 -mt-3" src={logo} />
-            <div className="text-white text-lg ml-3">
-              <div className="font-medium">Data Ocean</div>
-              <div className="intro-x text-white text-sm">
-                Platform
+            <a href="https://dataocean.us/" target="_blank" className="intro-x flex items-center pl-5 pt-4">
```
Comment (roman-tiukh) [inline src/components/pages/auth/LoginPage.jsx:52]: REACT_APP_LANDING_URL

File: src/locales/en/translation.json [hunk 2]
```diff
@@ -30,6 +30,7 @@
   "contactUs": "Contact us",
   "contacts": "Contacts",
   "continueWith": "Continue with",
+  "copyrights": "©2020 – Data Ocean. All rights reserved.",
```
Comment (roman-tiukh) [inline src/locales/en/translation.json:33]:
```suggestion
  "allRightsReserved": "All rights reserved",
```

**Review by roman-tiukh — CHANGES_REQUESTED** [review]

File: src/locales/en/translation.json (hunk 2 above)
Comment (Litsyshyn) [reply src/locales/en/translation.json:33]: "©2020 – Data Ocean"  -  should i leave it on page?

Comment (roman-tiukh) [reply src/locales/en/translation.json:33]: yes

**Comment by Litsyshyn** [comment]
Changed in assistance with Emma.

File: .env [hunk 3]
```diff
@@ -10,3 +10,4 @@ EXTEND_ESLINT=true
 
 REACT_APP_API_BASE_URL=https://ipa.dataocean.us
 REACT_APP_PEP_GRAPH_URL=https://dataocean.us/pep/
+REACT_APP_LANDING_URL=https://dataocean.us/
```
Comment (roman-tiukh) [inline .env:13]: ? this variable already in .env
https://github.com/3v-workspace/DataOcean/blob/master/.env

File: src/components/pages/auth/LoginPage.jsx [hunk 4]
```diff
@@ -14,6 +14,7 @@ import english from 'images/USA.svg';
 import ukrainian from 'images/ukraine.svg';
 import { setYupLanguage } from 'utils/setLanguage';
 
+const landingUrl = process.env.REACT_APP_LANDING_URL;
```
Comment (roman-tiukh) [inline src/components/pages/auth/LoginPage.jsx:17]: you dont need landingUrl variable

File: src/locales/en/translation.json [hunk 5]
```diff
@@ -30,6 +31,7 @@
...
```
Comment (roman-tiukh) [inline src/locales/en/translation.json:34]: delete this pls

File: src/locales/uk/translation.json [hunk 6]
```diff
@@ -30,6 +31,7 @@
   "contactUs": "Напишіть нам",
   "contacts": "Контакти",
   "continueWith": "Продовжити з",
+  "copyrights": "©2020 – Data Ocean. Всі права захищені.",
```
Comment (roman-tiukh) [inline src/locales/uk/translation.json:34]: delete this pls

File: src/components/pages/auth/SignInForm.jsx [hunk 7]
```diff
@@ -130,6 +130,7 @@ const SignInForm = () => {
           {t('privacyPolicy')}
           {/* {i18n.language === 'en' ? 'Українська' : 'English'} */}
         </a>
+        <p>©2020 – Data Ocean. {t('allRightsReserved')}</p>
```
Comment (roman-tiukh) [inline src/components/pages/auth/SignInForm.jsx:133]:
```suggestion
        <p>©2020 – Data Ocean. {t('allRightsReserved')}.</p>
```

File: src/locales/uk/translation.json (hunk 6 above)
Comment (roman-tiukh) [inline src/locales/uk/translation.json:34]:
```suggestion
  "copyrights": "©2020 – Data Ocean. Всі права захищені",
```

File: src/locales/uk/translation.json [hunk 8]
```diff
@@ -7,6 +7,7 @@
   "address": "Адреса",
   "afterRefreshTokenYouLoseAccess": "Після оновлення ви втратите доступ до системи з поточним токеном.",
   "all": "Всі",
+  "allRightsReserved": "Всі права захищені.",
```
Comment (roman-tiukh) [inline src/locales/uk/translation.json:10]:
```suggestion
  "allRightsReserved": "Всі права захищені",
```

**Review by roman-tiukh — CHANGES_REQUESTED** [review]

File: src/locales/uk/translation.json (hunk 6 above)
Comment (Litsyshyn) [reply src/locales/uk/translation.json:34]: Actually i was delete it, IDK why the translation still was there.

File: .env (hunk 3 above)
Comment (roman-tiukh) [inline .env:13]: ?

**Review by roman-tiukh — CHANGES_REQUESTED** [review]

**Review by roman-tiukh — APPROVED** [review]

**Pull request merged**